orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -178,15 +179,17 @@ def main():
 
     run_ortmodule_poc_net(cwd, log, no_cuda=False, data_dir=args.mnist)
 
-    run_ortmodule_poc_net(cwd, log, no_cuda=True, data_dir=args.mnist)
+    if os.getenv("ORTMODULE_ROCM_TEST", "0") == "0":
```

Comment:
What is the purpose of this?

Reply:
disable cpu training test. It will take nearly 15 mintues.

Reply:
This is not specific to rocm, can we rename it to `ORTMODULE_DISABLE_CPU_TEST` or something the like?

Reply:
Done.